athena-vertica/src/main/java/com/amazonaws/athena/connectors/vertica/VerticaMetadataHandler.java

```diff
@@ -367,11 +328,14 @@ public void getPartitions(BlockWriter blockWriter, GetTableLayoutRequest request
      */
     @Override
     public GetSplitsResponse doGetSplits(BlockAllocator allocator, GetSplitsRequest request)
-            throws SQLException
     {
```

Comment:
it doesn't seem we are handling splits from QPT? Is that on purpose?

Reply:
you could use: `setupQueryPassthroughSplit` from the parent class.

Reply:
here there is a somewhat different approach than other JDBC connectors. creating SQL statement from qpt args and exporting results to S3.